### DISCUSSION TITLE
Sonoff Basic freezes after update to 13.0

### DISCUSSION THREAD

**Discussion by Palisade8** [arendst/Tasmota#19108] [Q&A] [answered]
I've updated sonoff basic r2  (ESP8266EX) from 12.0.2.4 to 13.0 and it become freeze,  on version 12.4 freezes are also exist.   After downgrade to 12.0.2.3 it will work Fine.

Sonoff Basic r2 works with BMP280 sensor, disabling (solderiing out) sensor take no effect.

**Comment by sfromis** [comment, marked as answer]
When unsuitably using such bad devices as Sonoff Basic for adding stuff to, you were already living on the edge when it comes to it working, and minor differences may push it over. I'd only recommend that as an exercise in hunting down bad power problems, that thing can barely power itself.

In general, there is no "should" in upgrading to newer Tasmota versions, unless you want some specific improvement.

**Comment by Wintespe** [comment]
What's so bad about the Sonoff Basic?

**Reply by sfromis** [reply]
The quality. DC power supply too small, and tends to get worse with time. I've had two stop working, even without pushing them any way.
